apps/web/src/app/arrangementer/[slug]/[eventId]/tilbakemelding/page.tsx

```diff
@@ -0,0 +1,54 @@
+import { EventFeedbackForm } from "@/app/arrangementer/components/Feedback/FeedbackForm"
+import { auth } from "@/auth"
+import { server } from "@/utils/trpc/server"
+import { Text, Title } from "@dotkomonline/ui"
+import { formatDate } from "@dotkomonline/utils"
+
+const EventFeedbackPage = async ({ params }: { params: Promise<{ eventId: string }> }) => {
+  const { eventId } = await params
+  const session = await auth.getServerSession()
+  const event = await server.event.get.query(eventId)
+  const feedbackForm = await server.event.feedback.findFormByEventId.query(eventId)
+
+  if (!feedbackForm || !feedbackForm.isActive)
+    return <Text>Dette arrangementet har ikke et tilbakemeldingsskjema.</Text>
+
+  const user = session ? await server.user.getMe.query() : undefined
+  const attendees = event.attendanceId
+    ? await server.event.attendance.getAttendees.query({ attendanceId: event.attendanceId })
+    : []
+
+  const attendee = user && attendees?.find((attendee) => attendee.userId === user.id)
+  if (!attendee) return <Text>Du kan ikke svare på dette skjemaet.</Text>
+
+  const previousAnswer = await server.event.feedback.findAnswerByAttendee.query({
+    formId: feedbackForm.id,
+    attendeeId: attendee.id,
+  })
+
+  if (previousAnswer) return <Text>Du har allerede svart på dette skjemaet.</Text>
+
+  if (event.end.getTime() > Date.now())
```

Comment:
We should try to use date-fns as much as possible. Do you mind replacing with `isAfter(event.end, Date.now())`?